velox/parse/Expressions.h

```diff
@@ -87,33 +82,7 @@ class InputExpr : public core::IExpr {
     return EMPTY();
   }
 
-  std::shared_ptr<const IExpr> withInputs(
```

Comment:
Was this just legacy from Koski?

Reply:
That's right. This is a copy-paste from Koski.